e2e/src/preview-format.e2e-spec.ts

```diff
@@ -0,0 +1,42 @@
+import { WrapperPageObject } from './wrapper.po';
+import {
+  CHANGED_PREVIEW_FORMAT,
+  CHANGED_PREVIEW_VALUE,
+  DEFAULT_PREVIEW_FORMAT,
+  DEFAULT_PREVIEW_VALUE,
+  PREVIEW_FORMAT_WRAPPER
+} from './constants';
+
+
+/* Specific checks performed against the "Preview Format" duration picker */
+
+let page: WrapperPageObject;
+
+describe('Preview Format (specific checks)', () => {
+  beforeEach(async () => {
+    page = new WrapperPageObject(PREVIEW_FORMAT_WRAPPER);
+    await page.navigateTo();
+
+    page.dpDaysArrowUp.click();
+    page.dpDaysArrowUp.click();
+    page.dpHoursArrowUp.click();
+    page.dpMinutesArrowUp.click();
+    page.dpMinutesArrowUp.click();
+    page.dpMinutesArrowUp.click();
+    page.dpSecondsArrowUp.click();
+    page.dpSecondsArrowUp.click();
```

Comment:
why those clicks in the beforeEach ?

Reply:
I wanted to set some small value on the picker, and I put that in `beforeEach` because it is needed in both tests. I only now realized that I should have used `setValueButton` to handle that. I'd put it in `beforeEach` instead of this clicks if you're ok with that.